### PR TITLE
Issue 4178 percentage fee

### LINKED ISSUE #4178: Update Invoice Form For 10% Fee
Invoice form needs to be updated for 10% fee -- currently does not account for the 10% which goes to Gitcoin.

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description

Add Gitcoin platform fees to invoice form.

##### Refers/Fixes
Fixes #4178

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -1050,6 +1050,8 @@ def invoice(request):
     params['tips'] = [
         tip for tip in bounty.tips.send_happy_path() if ((tip.username == request.user.username and tip.username) or (tip.from_username == request.user.username and tip.from_username) or request.user.is_staff)
     ]
+    if (bounty.web3_created.replace(tzinfo=None) > date_to_start_charging_ten_percent.replace(tzinfo=None)):
+        params['percentage_fee'] = '10%'
```
Comment (thelostone-mc) [inline app/dashboard/views.py]: Ah so the percentage_fee changes per bounty as folks might get a discounted fee.
we've got a field in bounty model which stores `fee_amount` (but not %)

@danlipert what do we do here ?

Comment (octavioamu) [reply app/dashboard/views.py]: yep we have discounts associated to profiles and now cupons, so this should be dynamic

Comment (danlipert) [reply app/dashboard/views.py]: Yes we need to calculate what the percentage fee was for a given bounty, its not always 10%. I think we should use the `fee_amount` to figure it out

**Comment by danlipert** [comment]
@iamonuwa The testing field in the PR description is mandatory - please write how you tested your changes

**Comment by thelostone-mc** [comment]
@iamonuwa any update on this ?

**Comment by iamonuwa** [comment]
I'll finish this

**Comment by thelostone-mc** [comment]
@iamonuwa might be worth closing this PR if you won't be getting to it anytime soon ^_^

**Comment by thelostone-mc** [comment]
This has been ported over to https://github.com/gitcoinco/web/pull/5937
Thanks @iamonuwa  :)

**Pull request closed without merging**